background.ts

```diff
@@ -181,10 +181,17 @@ Electron.ipcMain.on('confirm-do-image-deletion', async(event, imageName, imageID
   });
 
   if (choice === 0) {
-    await imageManager.deleteImage(imageID);
-    refreshImageList();
+    try {
+      await imageManager.deleteImage(imageID);
+      refreshImageList();
+      event.reply('kim-process-ended', 0);
+    } catch (err) {
+      Electron.dialog.showMessageBox({
```

Comment:
Do we not need a reply in this case?

Reply:
I don't think so. It's all async. The Vue component isn't expecting a reply, but will act if it gets one.